src/styles/services-view.less

```diff
@@ -189,6 +191,7 @@ services-view,
       background-color: rgba(255,255,255,.9);
       border-radius: 2px 2px 0 0;
       display: inline-block;
+      float: left;
```

Comment:
Might be good to add a comment here as the combo of display: inline-block and float makes me suspicious.  Why is the float needed?